platforms/org.eclipse.mita.platform.xdk110/1.0.0/xdk110.platform

```diff
@@ -472,7 +478,18 @@ connectivity named-singleton BLE {
 	 * The name of the device as advertised via GAP.
 	 */
 	configuration-item deviceName : string
-	
+	/**
+	 * Boolean representing if the MAC address is to be configured
+	 */
+	configuration-item IsMacAddrConfigured : bool
+	/**
+	 * MAC Address to be configured. Unused if IsMacAddrConfigured is false
+	 */
+	configuration-item MacAddr : uint64
```

Comment:
- The combination of `IsMacAddrConfigured` and an optional `MacAddr` can be combined into just the MAC address. On the generator side you can differentiate between them by trying to get the mac adress; if the user didn't specify anything you will get null, otherwise the MAC address.
- please start configuration items lowercase and don't abreviate them unless the abreviation is common (like MAC).
- Configuring the MAC address with an uint64 seems cumbersome, is there some way this could be done using a string (like "00-80-41-ae-fd-7e" or "00:80:41:ae:fd:7e")?

Reply:
fixed